index.html

```diff
@@ -144,8 +149,8 @@ <h3><dfn>Term</dfn> interface</h3>
     <p>
       <dfn>termType</dfn> contains a value that identifies the concrete interface of the term, since
       Term itself is not directly instantiated. Possible values include <code>"NamedNode"</code>,
-      <code>"BlankNode"</code>, <code>"Literal"</code>, <code>"Variable"</code> and
-      <code>"DefaultGraph"</code>.
+      <code>"BlankNode"</code>, <code>"Literal"</code>, <code>"Variable"</code>, <code>"DefaultGraph"</code>,
+      <code>"Quad"</code> and <code>undefined</code>.
```

Comment:
I'm not a big fan of the `undefined` here.
By doing it this way, we propagate an RDF* concern to everywhere.

I would just leave it to the string values, and then on some interface declare it as optional.